pset-functions.js

```diff
@@ -136,6 +185,59 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+let problem3test = "javascript"
+console.log(`Test Word: ${problem3test}`)
+
+function scrabbleScore(word){
+
+  onePt = ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T']
+  twoPts = ['D', 'G']
+  threePts = ['B', 'C', 'M', 'P']
+  fourPts = ['F', 'H', 'V', 'W', 'Y']
+  fivePts = ['K']
+  eightPts = ['J', 'X']
+  tenPts = ['Q', 'Z']
+
+  words = word.split("")
+
+  let score = 0
+
+  words.forEach(function (element){
```

Comment:
@kertcodes nice logic, only issue is the structure of your `if` statement. The way your code is currently structured, each clause will be evaluated even if a match is found earlier on. 

for example: 

```
if (x === 1) {
} 

if (x === 2) { 
} 

if (x === 3) {
}
```
In the above code, *all* clauses would be evaluated even if x = 1 (and the first clause matched)

vs 

```
if (x === 1) {

} else if (x === 2) {

} else if (x === 3) {

} 
```
In the example above, if x = 1, the code would exit immediately after matching the first clause